sparse/usr/sbin/flash-partition

```diff
@@ -199,6 +199,10 @@ if on_device; then
                 fi
         fi
 
+	if [[ "$SAILFISHOS_WIPE_PARTITIONS" == "1" ]]; then
+		dd if=/dev/zero bs=4096 of=$DEVDIR/$DEVICE
```

Comment:
I would say at least bs=1M. This could speed things up because of lowering number of syscalls and writing more data at once.